app/views/posts/index.html.erb

```diff
@@ -3,9 +3,27 @@
   <head>
     <title>GApp</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
+	<link rel="stylesheet" type="text/css" href="application.css">
   </head>
   <body>
-    <h1>List of all posts</h1>
-<p>Placeholder: Here is a list of posts for a given user.</p>
- </body>
+  <h1>List of Posts</h1>
+
+<div class="post-list">
+  <% @posts.each do |post| %>
+    <div class="container">
+      <h2><%= post.title %></h2>
+      <p><%= post.text %></p>
+      <div class="comments">
+        <p><%= post.comments.count %> Comments</p>
+        <ul>
+          <% post.comments.each do |comment| %>
+            <li><%= comment.text %></li>
+          <% end %>
+        </ul>
+      </div>
+    </div>
+  <% end %>
+</div>
+</div>
+   </body>
 </html>
```

Comment:
- [ ] Again kindly fix your design to follow the given wireframe.

Your design
![image](https://github.com/Gardimy/BlogApp/assets/81932478/b9ff870e-5018-474b-9a7d-6de69421763d)


Given design
![image](https://github.com/Gardimy/BlogApp/assets/81932478/8f460493-b934-49cd-9510-1680b27374b1)


- The image should be present
- Username and number of posts should be present
- Each post should show comments and a number of likes


- [ ] Here we should use the `post.recent_comments` method created in the Post model to render 5 most recent comments in the `app/views/posts/index.html.erb` page